app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/general-settings.gjs

```diff
@@ -1,37 +1,76 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { on } from "@ember/modifier";
 import { action } from "@ember/object";
 import DButton from "discourse/components/d-button";
 import DEditor from "discourse/components/d-editor";
 import UppyImageUploader from "discourse/components/uppy-image-uploader";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 import i18n from "discourse-common/helpers/i18n";
 
 export default class AdminConfigAreasAboutGeneralSettings extends Component {
+  @tracked showSavedAlert = false;
+
+  name = this.args.generalSettings.title.value;
+  summary = this.args.generalSettings.siteDescription.value;
+  extendedDescription = this.args.generalSettings.extendedSiteDescription.value;
+  aboutBannerImage = this.args.generalSettings.aboutBannerImage.value;
+
+  @action
+  async save() {
+    this.showSavedAlert = false;
+    try {
+      await ajax("/admin/config/about.json", {
+        type: "PUT",
+        data: {
+          general_settings: {
+            name: this.name,
+            summary: this.summary,
+            extended_description: this.extendedDescription,
+            about_banner_image: this.aboutBannerImage,
+          },
+        },
+      });
+      this.showSavedAlert = true;
```

Comment:
Ditto ^^ Might be worth moving into a `finally`?

Reply:
Yes I agree with Regis here.

Reply:
Actually we won't need them if we use toasts instead